engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -89,10 +89,19 @@ <h4 translate>Subscriptions</h4>
 
       <dl class="dl-horizontal dl-horizontal-left">
         <dt translate>Subscription Status</dt>
-        <dd>
+        <dd ng-if="!simpleContentAccessEnabled">
           <i ng-class="getHostStatusIcon(host.subscription_global_status)"></i>
           {{ host.subscription_status_label }}
         </dd>
+        <dd ng-if="simpleContentAccessEnabled">
+          <!-- always display green icon for SCA -->
+          <i
+            ng-class="getHostStatusIcon(0)"
+            title = "{{ 'This organization has Simple Content Access enabled. Hosts can consume from all repositories in their Content View regardless of subscription status.' | translate }}"
+            >
+          </i>
+          {{ 'Simple Content Access' | translate }}
```

Comment:
I'm tempted to handle this change in the subscription_status.rb to push it further back in the stack - and so it'll give a more meaningful result in the API, and hammer. See what it looks like now, with errata status included for comparison:

```
"errata_status": 1,
"errata_status_label": "Could not calculate errata status, ensure host is registered and the katello-host-tools package is installed",
"subscription_status": 5,
"subscription_status_label": "Disabled",
```

IOW I think we can change the label to reflect the messaging you've provided here.

Reply:
Or maybe it can become just 'Simple Content Access' rather than the full message? Whichever feels right

Reply:
Is `Katello::Candlepin::Consumer::ENTITLEMENTS_DISABLED` used _only_ for SCA?

Reply:
Yep!

Reply:
Thanks, updating the label in `subscription_status.rb` to read "Simple Content Access" instead.